certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json

```diff
@@ -437,19 +450,23 @@
                     "properties": {
                       "file_name": {
                         "type": "string",
-                        "description": "file_name"
+                        "description": "file_name",
+                        "title": "file name"
                       },
                       "diseases_found": {
                         "type": "string",
-                        "description": "diseases_found"
+                        "description": "diseases_found",
+                        "title": "diseases found"
                       },
                       "cpt_code": {
                         "type": "string",
-                        "description": "cpt_code"
+                        "description": "list of cpt codes",
+                        "title": "cpt code"
                       },
                       "cpt_desc": {
                         "type": "string",
-                        "description": "cpt_desc"
+                        "description": "cpt_desc",
```

Comment:
@gayathri35 All the description values of properties should be free from grammatical and spell errors. please provide valid description values for all the properties.